schemas/iso19139/src/main/plugin/iso19139/formatter/eu-geodcat-ap-semiceu/view.xsl

```diff
@@ -2694,7 +2694,14 @@
 
   <xsl:template mode="iso19139-to-dcatap" name="Dates" match="gmd:date/gmd:CI_Date">
     <xsl:param name="date">
+      <xsl:choose>
+        <xsl:when test="gmd:date/gco:Date">
       <xsl:value-of select="normalize-space(gmd:date/gco:Date)"/>
+        </xsl:when>
+        <xsl:when test="gmd:date/gco:DateTime">
+          <xsl:value-of select="normalize-space(gmd:date/gco:DateTime)"/>
+        </xsl:when>
+      </xsl:choose>
```

Comment:
Can be simplified with?

```
 <xsl:value-of select="normalize-space(gmd:date/*)"/>
```
 
 or
 
 ```
 <xsl:value-of select="normalize-space(gmd:date/(gco:Date|gco:DateTime))"/>
 ```

Reply:
We should probably keep the changes to the SEMIC conversion to the minimum and propose change upstream if needed. It will facilitate future updates.